apps/etterlatte-saksbehandling-ui/client/src/components/vurderingsboks/VurderingsboksWrapper.tsx

```diff
@@ -0,0 +1,125 @@
+import { BodyShort, Button, Heading } from '@navikt/ds-react'
+import { Delete, Edit } from '@navikt/ds-icons'
+import React, { ReactElement, useState } from 'react'
+import styled from 'styled-components'
+
+interface Props {
+  tittel: string
+  subtittelKomponent?: ReactElement
+  oppfyltUnntaksvilkaarstittel?: string
```

Comment:
ser for meg at vi kunne brukt denne komponenten overallt der vi skal fylle in eller vurdere informasjon til bruker. men føler att unntaktsvilkaar er veldig knyttet mot vilkårsvurderingen.